gen/generator.go

```diff
@@ -374,3 +383,23 @@ func (SnakeCaseFieldNamer) GetJSONFieldName(t reflect.Type, f reflect.StructFiel
 
 	return camelToSnake(f.Name)
 }
+
+// CamelCaseFunctionNamer implements FunctionNamer interface with CamelCase format
+type CamelCaseFunctionNamer struct{}
+
+func (CamelCaseFunctionNamer) GetName(keepFirst bool, parts ...string) string {
```

Comment:
I'd make it a simple function (not a function on a struct) and call it directly rather than via an interface. I don't think we'll ever need any extensibility in naming auxiliary internal functions.